src/util/rsthreads.cc

```diff
@@ -227,7 +242,7 @@ RsQueueThread::RsQueueThread(uint32_t min, uint32_t max, double relaxFactor )
     :mMinSleep(min), mMaxSleep(max), mRelaxFactor(relaxFactor)
 {
     mLastSleep = (uint32_t)mMinSleep ;
-    mLastWork = time(NULL) ;
+    mLastWork = 1000 * getCurrentTS() ;
```

Comment:
it would be helpful to rename mLastWork into mLastWork_ms, so that the units are obvious when reading the code.

Reply:
agreed